lib/SPIRV/libSPIRV/spirv_internal.hpp

```diff
@@ -52,10 +52,7 @@ enum InternalSourceLanguageNonSemanticDI {
   ISourceLanguageCPP20 = 219,
 };
 
-enum InternalLinkageType {
-  ILTPrev = LinkageTypeMax - 2,
-  ILTInternal
-};
+enum InternalLinkageType { ILTPrev = LinkageTypeMax - 2, ILTInternal };
```

Comment:
While this suggesting comes from clang-format, probably enums should be better to be split into multiple lines. I'll add ignore format to this file I guess